test/utils/executor.hpp

```diff
@@ -61,8 +61,7 @@ std::shared_ptr<ExecType> init_executor(
 
 template <>
 inline std::shared_ptr<gko::ReferenceExecutor>
-    init_executor<gko::ReferenceExecutor>(
-        std::shared_ptr<gko::ReferenceExecutor>)
+init_executor<gko::ReferenceExecutor>(std::shared_ptr<gko::ReferenceExecutor>)
```

Comment:
should we adjust the setting to indent this situation?

Reply:
I think this is fine (and also will be changed in #1236)